<?xml version="1.0" encoding="UTF-8"?>
<problems>
  <problem>
    <file>test.py</file>
    <line>9</line>
    <description>Exception class 'Exception' has already been caught</description>
  </problem>
  <problem>
    <file>test.py</file>
    <line>16</line>
    <description>Exception class 'Exception' has already been caught</description>
  </problem>
</problems>